src/analysis/inner_product_space/basic.lean

```diff
@@ -740,10 +743,10 @@ begin
   split,
   { intros h v hv w hw,
     convert h ⟨v, hv⟩ ⟨w, hw⟩ using 1,
-    simp },
+    simp only [subtype.mk_eq_mk] },
```

Comment:
What's the goal state here? This feels pretty weird, does `refl` close the goal?